packages/astro/package.json

```diff
@@ -214,8 +214,11 @@
     "srcset-parse": "^1.1.0",
     "unified": "^10.1.2"
   },
+  "peerDependencies": {
+    "sharp": "^0.32.1"
+  },
   "engines": {
     "node": ">=18.14.1",
-    "npm": ">=6.14.0"
```

Comment:
This will cause a warning for everyone using yarn and npm, won't it?

Reply:
Do we get a custom release that I can test against? I don't think this shall show warning for the other packages. But happy to test it out.

Reply:
See this PR to the Prettier plugin we had a few weeks ago that explicitly removed pnpm from engines because it caused a warning: https://github.com/withastro/prettier-plugin-astro/pull/362

Reply:
Oh that's a learning! Thanks @Princesseuh